### DISCUSSION TITLE
Rendering polybar per workspace

### DISCUSSION THREAD

**Discussion by M-Mabrouk1** [leftwm/leftwm#717] [General]
I was wondering if I have and ultrawide screen and I have more than workspace,
how can I render polybar per workspace not per monitor,
I played a little with the last few lines in the up script but I couldn't get it to work.

**Comment by VuiMuich** [comment]
Iirc the `basic_polybar` should do exactly that, no?

**Reply by M-Mabrouk1** [reply]
no the basic polybar doesn't render any polybars for me, my usual config renders it per monitors, I couldn't get it to work per workspace on one monitor.
